### PR TITLE
10833 notifications text

### PR DESCRIPTION
**Pull request by elwoodva**
## Description
Changed the text of the success message to match the mockup.

## Screenshots

Screenshot 2019-10-02 at 9 40 40 AM

### CODE REVIEW

File: src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx [hunk 1]
```diff
@@ -133,7 +134,8 @@ export function mapStateToProps(state, ownProps) {
     hasError ||
     isPending;
   const transactionSuccess =
-    state.vet360.transactionStatus === 'COMPLETED_SUCCESS';
+    state.vet360.transactionStatus ===
+    VET360.TRANSACTION_STATUS.COMPLETED_SUCCESS;
```
Comment (rianfowler) [inline src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx]: Maybe this branch has unrelated changes in it?

**Review by rianfowler — COMMENTED** [review]
Please tag the @department-of-veterans-affairs/frontend-review-group in a new comment when this has at least one approval from a teammate. We suggest doing peer reviews with draft PRs. More info here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/code_review_guidelines.md

File: src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx (hunk 1 above)
Comment (elwoodva) [reply src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx]: I was just correcting some magic strings that were pointed out in the last pull request.

**Review by talbotmj — APPROVED** [review]

**Comment by elwoodva** [comment]
@department-of-veterans-affairs/frontend-review-group

**Review by rianfowler — APPROVED** [review]
